packager/media/codecs/avc_decoder_configuration_record_unittest.cc

```diff
@@ -67,9 +67,10 @@ TEST(AVCDecoderConfigurationRecordTest, SuccessWithSPSExtension) {
       0x01,  // num_pps
         0x00, 0x06,  // size
         0x68, 0xEB, 0xE3, 0xCB, 0x22, 0xC0,
-        0x03, 0x5A, 0x2C, // sps extension first 24 bits are chroma_format, 
-        // bit_depth_luma_minus8, bit_depth_chroma_minus8
-        0x01, // num_sps_ext 
+      0xFC, // Least significant 2 bits is chroma_format 
```

Comment:
Google C++ style guide requires at least two spaces before the comment. Same below.

```c++
      0xFC,  // Least significant 2 bits is chroma_format 
```

Reply:
done